lib/wovnrb.rb

```diff
@@ -35,6 +35,13 @@ def call(env)
       default_lang = @store.settings['default_lang']
       return @app.call(env) if @store.settings['test_mode'] && @store.settings['test_url'] != headers.url
 
+      cookie_lang = Rack::Request.new(env).cookies['wovn_selected_lang']
+      request_lang = headers.lang_code
+      if @store.settings['use_cookie_lang'] && request_lang != cookie_lang && request_lang == @store.default_lang
```

Comment:
I'm not sure which case we need condition `request_lang == @store.default_lang`.

Reply:
I based it from WOVN.php's implementation (see [here](https://github.com/WOVNio/WOVN.php/blob/13b41a8fa8988388a88304cef95774e69edcb99a/src/wovnio/wovnphp/Headers.php#L377-L386)) but yeah I have my doubts on it as well.

I think we can remove it, since I believe the idea is that "if the cookie is set, and the request language is not the same as the cookie, then we redirect _regardless of what the request language is_".

Reply:
You can read the original discussion I had in https://wovnio.atlassian.net/browse/ST-508

Reply:
Thanks, makes sense. Let's keep the condition as-is then.